src/codegen/jsonschema/additionalproperties.rs

```diff
@@ -30,10 +30,10 @@ pub fn from_object_with_additional_properties(
                     model: Box::new(model?),
                 })))
             }
-            Value::Bool(_) => Err(Error::SchemaInvalidProperty(
-                // todo: bool support maybe as a flag
-                "additionalProperties".to_string(),
-            )),
+            Value::Bool(true) => Ok(Model::new(ModelType::MapType(MapType {
```

Comment:
i'm wondering if we can make here ObjectType with empty properties but additional = true,
same would be in 41 line